docs/guides/4f-js-rulesets.md

```diff
@@ -0,0 +1,110 @@
+## JavaScript Ruleset Format
+
+Spectral v6.0 added support for a JavaScript ruleset format, similar to the JSON and YAML formats.
```

Comment:
I think I asked before. Do we need the v6.0 here?

Reply:
You did, and I tagged other folks in that comment but didn't get a reply, so it stayed in there since I wasn't sure if it was ok to remove.